src/components/Form/RadiobuttonGroup.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from 'react';
+import classnames from 'classnames';
+import { HtmlDiv, HtmlSpan } from '../../reset';
+import { VisuallyHidden } from '../Visually-hidden/Visually-hidden';
+import { idGenerator } from '../../utils/uuid';
+
+import { RadiobuttonProps } from './Radiobutton';
+
+const baseClassName = 'fi-radiobuttongroup';
+const radiobuttonGroupClassNames = {
+  container: `${baseClassName}_container`,
```

Comment:
Is this `_container` class useless? Should we remove it?

Reply:
Seems to be the only classname that actually point to the div containing radiogroup? In the actual Radiobutton.tsx it seems to be redundant since it's only used for the same element with baseClassName?

Reply:
> Seems to be the only classname that actually point to the div containing radiogroup?

True that. It is the only classname that has the children, **Radiobuttons** or/and **RadiobuttonDividers**
So it can be left as now.

Reply:
> In the actual Radiobutton.tsx it seems to be redundant since it's only used for the same element with baseClassName?

Do you mean that we should remove the `baseClassName` from the **components** side and just add that in the **core** side?

Reply:
Nope, I'd remove the container here since the element with baseClass name is the container.

Reply:
So you want to remove the `fi-radiobuttongroup_container` then? Because if removed, then there would be no specific classname provided to catch the container for radios.
Or would it better to rename it to reflect it's use better, like `fi-radiobuttongroup_radios`?

Because the **baseClassName**,  `fi-radiobuttongroup`,  is only provided to root of the component.

Reply:
Ah, sorry for misunderstanding. For the buttonGroup current implementation seems to be solid, but for the components/Form/Radiobutton.tsx there is also a container class which in my opinion is redundant.

Reply:
Seems to be ok now. There hierarchy is as follows: fi-radiobuttongroup_container > fi-radiobutton_container > fi-radiobutton-input & fi-radiobutton_label and so on.